src/EnergyPlus/StandardRatings.cc

```diff
@@ -4606,7 +4606,7 @@ namespace StandardRatings {
                             std::tie(q, e, NetTotCoolCapBinned_2023, TotCoolElecPowerBinned_2023) = IntermediateCapacityAndPowerSEER2Case2A(
                                 p_int, q_int, q_low, bl, n, Q_E_Int(spnum), q_full, P_E_Int(spnum), p_full, p_low);
                             goto SpeedLoop3_exit;
-                        } else if (bl < q_full) {
+                        } else { // bl < q_full
```

Comment:
[src/EnergyPlus/StandardRatings.cc:4609]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.
[